pkg/controllers/podopslifecycle/utils.go

```diff
@@ -76,7 +77,8 @@ func IDToLabelsMap(pod *corev1.Pod) (map[string]map[string]string, map[string]in
 }
 
 // IsLifecycleOnPod returns true if the lifecycle with lifecycleId exist on pod, otherwise returns false
-func IsLifecycleOnPod(lifecycleId string, pod *corev1.Pod) (bool, error) {
+func IsLifecycleOnPod(lifecycleId string, obj client.Object) (bool, error) {
```

Comment:
Use `pod *corev1.Pod` is more suitable?

Reply:
[IsLifecycleOnPod is called by ](https://github.com/KusionStack/kuperator/pull/295/commits/947ca3c8b1c14af6f76c5052a132f59718c472a2)